test/common/plugin/currencyPlugin.fixtures/currencies/litecoin.ts

```diff
@@ -0,0 +1,272 @@
+import { FixtureType, key } from '../common'
+
+export const litecoin: FixtureType = {
+  pluginId: 'litecoin',
+  WALLET_TYPE: 'wallet:litecoin',
+  WALLET_FORMAT: 'bip44',
+  'Test Currency code': 'LTC',
+  key,
+  xpub:
+    'xpub6CUgpZYSditAHLGHepYbZzskB16AQx69mnD5iUVjd7pdWjP67ZzMNCw8n8Sg2xidi2pVAd35rbeYjxSr5CMUfEGgggFr8Z4bwm1n8YpV8Rn',
+  'invalid key name': {
+    id: 'unknown',
+    type: 'wallet:litecoin',
+    keys: {
+      litecoinKeyz: '12345678abcd',
+      format: 'bip44'
+    }
+  },
+  'invalid wallet type': {
+    id: 'unknown',
+    type: 'shitcoin',
+    keys: { litecoinKeyz: '12345678abcd' }
+  },
+  parseUri: {
+    'address only': [
+      'LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T',
+      {
+        publicAddress: 'LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T',
+        metadata: {}
+      }
+    ],
+    'address only2': [
+      'MSS1jxX7vEjHi5ujzszwTsZCnSDhkWKrBd',
+      {
+        publicAddress: 'MSS1jxX7vEjHi5ujzszwTsZCnSDhkWKrBd',
+        metadata: {}
+      }
+    ],
+    'uri address': [
+      'litecoin:LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T',
+      {
+        publicAddress: 'LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T',
+        metadata: {}
+      }
+    ],
+    'uri address with amount': [
+      'litecoin:LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T?amount=12345.6789',
+      {
+        publicAddress: 'LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T',
+        metadata: {},
+        nativeAmount: '1234567890000',
+        currencyCode: 'LTC'
+      }
+    ],
+    'uri address with amount & label': [
+      'litecoin:LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T?amount=1234.56789&label=Johnny%20Litecoin',
+      {
+        publicAddress: 'LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T',
+        metadata: {
+          name: 'Johnny Litecoin'
+        },
+        nativeAmount: '123456789000',
+        currencyCode: 'LTC'
+      }
+    ],
+    'uri address with amount, label & message': [
+      'litecoin:LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T?amount=1234.56789&label=Johnny%20Litecoin&message=Hello%20World,%20I%20miss%20you%20!',
+      {
+        publicAddress: 'LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T',
+        metadata: {
+          name: 'Johnny Litecoin',
+          notes: 'Hello World, I miss you !'
+        },
+        nativeAmount: '123456789000',
+        currencyCode: 'LTC'
+      }
+    ],
+    'uri address with unsupported param': [
+      'litecoin:LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T?unsupported=helloworld&amount=12345.6789',
+      {
+        publicAddress: 'LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T',
+        metadata: {},
+        nativeAmount: '1234567890000',
+        currencyCode: 'LTC'
+      }
+    ],
+    'legacy address only': [
+      '3EUxVRMAwPk2ZpBdiqRtY9uxWXSkgVCTFN',
+      {
+        publicAddress: 'MLh6oJm8tWbTNKTXpiREMoAMqE3CbYzgi6',
+        metadata: {}
+        // legacyAddress: '3EUxVRMAwPk2ZpBdiqRtY9uxWXSkgVCTFN'
```

Comment:
This legacy address should be supported.

Reply:
Re: @thehobbit85 's feedback. I don't agree with #1 or #2. One large fixture file is hard to maintain. Json is no easier to work with than a typescript .ts file. JSON is generally harder to work with since you need to manually type quotes for all keys.